bigtop-packages/src/charm/mahout/layer-mahout/README.md

```diff
@@ -0,0 +1,94 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+## Overview
+
+
+The Apache Mahout™ project's goal is to build an environment for quickly creating
+scalable performant machine learning applications.
+
+Apache Mahout software provides three major features:
+ * A simple and extensible programming environment and framework for building scalable algorithms
+ * A wide variety of premade algorithms for Scala + Apache Spark, H2O, Apache Flink
+ * Samsara, a vector math experimentation environment with R-like syntax which works at scale
+
+
+## Usage
+
+This charm is intended to be added to an existing cluster
+for example a Hadoop one. A Hadoop cluster can be deployed via
```

Comment:
typo: 'this charm is intended to be added to an existing cluster - e.g., hadoop. ...'